src/main/java/com/epam/parso/impl/SasFileParser.java

```diff
@@ -508,13 +508,23 @@ private String findCompressionLiteral(String src) {
     }
 
     /**
-     * The function to read next row from current sas7bdat file.
+     * The function to read and process all columns of next row from current sas7bdat file.
      *
-     * @param columnNames list of column names which should be processed.
      * @return the object array containing elements of current row.
      * @throws IOException if reading from the {@link SasFileParser#sasFileStream} stream is impossible.
      */
-    Object[] readNext(List<String> columnNames) throws IOException {
```

Comment:
I think if we are making SasFileParser public, we need to change some JavaDocs (?) that refer to SasFileParser as "internal" class. We might want to mention this issue 51

Reply:
JavaDocs for SasFileParser and SasFileReaderImpl were fixed.